capi/Makefile.am

```diff
@@ -17,6 +17,7 @@ libyami_capi_ldflags = \
 
 libyami_capi_cppflags = \
 	$(LIBVA_CFLAGS) \
+	-I$(top_srcdir)/interface \
```

Comment:
This might be done another way.  I really do not like the directory layout currently.  Things that install in include/libyami come from all over the project directories.  Why not have an include directory in the project so the header use is the same in the project as outside the project.